tests/test_modernize_openedx_yaml.py

```diff
@@ -0,0 +1,38 @@
+import os
+import shutil
+import uuid
+from unittest import TestCase
+from ruamel.yaml import YAML
+
+from edx_repo_tools.modernize_openedx_yaml import YamlModernizer
+
+
+class TestYamlModernizer(TestCase):
+
+    def setUp(self):
+        self.yaml_file = TestYamlModernizer._setup_local_copy("sample_openedx.yaml")
+
+    @staticmethod
+    def _setup_local_copy(file_name):
+        current_directory = os.path.dirname(__file__)
+        temp_file = os.path.join(current_directory, str(uuid.uuid4()) + ".yaml")
```

Comment:
This works fine, but in the future you may want to consider the [tmpdir](https://docs.pytest.org/en/stable/tmpdir.html) pytest fixture; probably a fair bit simpler to implement that way.  Don't need to manually invent a filename, automatically handles removing the file at the end of the test, etc.